example/docker/infrasim/start_node.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+CONTAINER=infrasim_1
+INTERFACE=eth1
+BRIDGE=br0
+
+# create the infrasim container
+docker create -it --net=docker_southbound --privileged --name $CONTAINER infrasim/infrasim-compute:3.5.1
```

Comment:
if you use ```-p xxxx:5901``` , you can start multiple infrasim docker.
for infrasim #1, use ```-p 15901:5901```
for infrasim #2, use ```-p 15902:5901``` ...etc